nvOC

```diff
@@ -219,20 +219,14 @@ Restart()
 ############################### Start nvOC ####################################
 Start()
 {
-   pkill -f gnome-terminal
-   pkill -f 5watchdog
-   pkill -f 7telegram
-   pkill -f 6tempcontrol
-   pkill -e screen
-   pkill -f 8wtm
-   if [ $LOCALorREMOTE == "LOCAL" ]
-   then
-     gnome-terminal --window-with-profile=mining
-     echo -e "${B}Miner Restart....${N}"; echo ""
-   fi
    if [ $LOCALorREMOTE == "REMOTE" ]
    then
      export DISPLAY=:0
+   fi
+   if ps ax | grep "[g]nome-terminal"
+   then
+     echo "nvOC already running, use ./nvOC restart"
+   else
      gnome-terminal --window-with-profile=mining
      echo -e "${B}Miner Restart....${N}"; echo ""
    fi
```

Comment:
This 'fi' should be at the and of the function, after kill commands

Reply:
But there is no kill command any more, it just checks if gnome-terminal is running or not and start if not.